processor/elasticinframetricsprocessor/processor.go

```diff
@@ -28,6 +28,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const OTelRemappedLabel = "otel_remapped"
```

Comment:
```suggestion
const OTelRemappedLabel = common.OTelRemappedLabel
```

Reply:
Sorry this does not work!

Reply:
What do you mean? Are not able to import `github.com/elastic/opentelemetry-lib/remappers/common`?

Reply:
It was not there this lib. I imported and now all ok!